AppCenterAnalytics/AppCenterAnalytics/MSAnalytics.m

```diff
@@ -187,6 +187,25 @@ + (BOOL)isAutoPageTrackingEnabled {
 
 #pragma mark - Private methods
 
+- (BOOL)validateLog:(MSLogWithNameAndProperties *)log {
```

Comment:
I would say we can get rid of this method and do everything in the shouldFilter.